prime-dotnet-webapi/Services/EmailTemplateService.cs

```diff
@@ -41,5 +42,24 @@ public async Task<EmailTemplateViewModel> GetEmailTemplateAsync(int id)
                 .ProjectTo<EmailTemplateViewModel>(_mapper.ConfigurationProvider)
                 .SingleOrDefaultAsync();
         }
+
+        public async Task<EmailTemplateViewModel> UpdateEmailTemplateAsync(int id, string template)
+        {
+            var emailTemplate = await _context.EmailTemplates.Where(t => t.Id == id).SingleOrDefaultAsync();
```

Comment:
this can be simplified to 
```C#
var emailTemplate = await _context.EmailTemplates
    .SingleOrDefaultAsync(t => t.Id == id);
```

Reply:
So just for clarity, Does that not do the compare on the server side. When the where does it DB side?